src/main/java/Buddy.java

```diff
@@ -0,0 +1,158 @@
+import java.util.Scanner;
+
+public class Buddy {
+    public static void main(String[] args) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Im Shrek");
+        printShrekFace2();
+        System.out.println("what are you doing in my swamp!??");
+        System.out.println("____________________________________________________________");
+
+        Scanner scanner = new Scanner(System.in); //Scanner object named "scanner"
+        Task[] tasks = new Task[100]; //Array of "task" object to store the tasks
+        int taskCount = 0; //Counter for the number of tasks
+        String input;
+
+        while (true) {
+            input = scanner.nextLine();
+
+            if (input.equals("bye")) //end loop when user inputs "bye"
```

Comment:
You could make the input handling a seperate method, since your main is really big.